result.go

```diff
@@ -15,9 +15,8 @@ import "database/sql/driver"
 // This is accessible by executing statements using sql.Conn.Raw() and
 // downcasting the returned result:
 //
-//    res, err := rawConn.Exec(...)
-//    res.(mysql.Result).AllRowsAffected()
-//
+//	res, err := rawConn.Exec(...)
```

Comment:
this indentation change is weird. gofmt problem?

Reply:
https://go.dev/doc/go1.19#go-doc

Since go 1.19, gofmt correct some format in comments.